ui/components/app/mmi/interactive-replacement-token-notification/interactive-replacement-token-notification.container.js

```diff
@@ -0,0 +1,43 @@
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router-dom';
+import { compose } from 'redux';
+import { getMMIActions } from '../../../../store/actions';
+import { getCurrentKeyring } from '../../../../selectors';
+import InteractiveReplacementTokenNotification from './interactive-replacement-token-notification.component';
+
+const mapStateToProps = (state) => {
```

Comment:
We no longer use the .component / .container format, we use functional components that make use of useSelector / useDispatch. Please refactor before this can merge.